go/vt/worker/multi_split_diff.go

```diff
@@ -0,0 +1,691 @@
+/*
+Copyright 2017 Google Inc.
```

Comment:
i think i asked this in slack at some point as well.
are we still doing copyright like this? probably at least want to bump the year, but i don't remember what the word was on the entity to assign to

Reply:
Year needs to be bumped. 
We need to change Google to "The Vitess Authors", like here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubectl/kubectl.go. But I don't see a contributors file. Maybe that's not needed? @caniszczyk can clarify.

Reply:
yes please change it to "The Vitess Authors" moving forward